doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
 Traceability to Reference Model.
 
 <a name="2.3"></a>
-## 2.3 Requirements.
+## 2.3 Containerised Platform Requirements.
+
+The Reference Model (RM) defines the NFVI to consist of the physical resources, virtualised resources and a software managment system.  In the virtualised world, the NFVI Software Management System (Virtualised Infrastructure Manager -- VIM) consists of the Guest Operating System, Hypervisor and, if needed, other software such as libvirt.  And the VIM is responsible for, among others, tenant management, resources management, inventory, scheduling, and access management. Now consider, the containerisation equivalent.
+
+Containerised Infrastructure Manager (CIM): a functional block that is responsible for controlling and managing the containerised NFVI compute, storage and network resources. Note CIM supports the Life Cycle Management of conatinerised and the underlying physical resources.
 
 <a name="2.3.1"></a>
 ### 2.3.1 General
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.gen.ost.01` | Open source | The Architecture **must** use Kubernetes APIs.|
-| `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through Kubernetes APIs. |
+| `req.gen.k8s.01` | Open source | The Architecture **must** use Kubernetes APIs.|
+| `req.gen.k8s.02` | Open source | The Architecture **must** support dynamic request and configuration of resources (compute, network, storage) through Kubernetes APIs. |
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. |
+| `req.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. |
+| `req.gen.scl.01` | Scalability | The Architecture **should** support policy driven horizontal auto-scaling. |
```

Comment:
Of VNF-s or the NFVI?

Reply:
I read "the Architecture" as "the Architecture of the NFVI". If this is how it is, maybe there should be a clarification (or we should just spell it out everywhere, so requirements can be quoted without ambiguity. My2c anyway.

Reply:
Okay. My reading was about the scaling of the VNF-s, so we deffinetly need a clarification. 
Isn't auto scaling of the infrastructure is difficult as in some point it will involve persons with 🛠️-s.

Reply:
Very true :-)

Reply:
horizontal scaling is IMO mostly used by VNFs
elasticity in the context of an NFVI is usually about scaling computes

an additional requirement for automated Node-scaling? as this RQ is basically a reference to Kubernetes HPA, the new requirement could be a reference to Node autoscaling: https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler

Reply:
I propose to change 'The Architecture **should** support policy driven horizontal auto-scaling. ' to 'The Architecture **should** support policy driven horizontal auto-scaling of VNF-s. '